docs/new-environments.md

```diff
@@ -0,0 +1,163 @@
+# Setting Up An Environment For RMI
+
```

Comment:
Could you also document the changes that were made to resolve the issue which was going on for a while with regard to sign-in?

Reply:
the 500 error and connection refused error, which we came up when we signed in, has been addressed in the page (see the network policy section). I will add a small bit on the end though, about adding a url to the allow lists (but this doesn't normally have to be done when setting up an environment as we usually dont create new urls)